playbooks/roles/openconnect/tasks/firewall.yml

```diff
@@ -12,16 +12,15 @@
     proto: "any"
     rule: "allow"
 
-- name: Enable NAT
-  command: "{{ ocserv_firewall_rule }}"
-
-- name: "Add Openconnect firewall persistence service to init"
+- name: Install the ocserv iptables service file
   template:
-    src: streisand-openconnect-service.sh.j2
-    dest: /etc/init.d/streisand-openconnect
-    mode: 0755
+    src: ocserv-iptables.service.j2
+    dest: /etc/systemd/system/ocserv-iptables.service
+    mode: 0644
 
-- name: "Enable the streisand-openconnect init service"
-  service:
-    name: streisand-openconnect
+- name: Enable the ocserv-iptables service
```

Comment:
I have the distinct memory of finding that the `systemd` task didn't handle specifying `enabled` and `state` at the same time very well. It seems like this works correctly in CI so perhaps I'm remembering wrong?

Reply:
That could be, however running it against a real server seemed to work fine in my case. Don't really have an issue with making them separate either.

Reply:
I think its fine to leave as-is. I'm not sure why I thought it was useful to comment with superstitious observations :laughing: